cypress/integration/LoggedOutStaticFAQ.js

```diff
@@ -5,8 +5,12 @@ describe('Display Static FAQ', () => {
   it('User can navigate into FAQ Article', () => {
     cy.visit('/');
     cy.get('[data-cy=btn-nonexistent-booking]').trigger('click');
-    cy.get('.signInOrBack div').contains('Sign In');
-
+    cy.get('[data-cy=sign-in-link]').should($signInLink => {
+      // F.I.X.M.E: TextLink Orbit Component is breaking text with space to multiple lines
```

Comment:
`F.I.X.M.E` => `F.I.X.M.E` - eslint warning is such case is fine, at least we now it should be fixed, otherwise no one bat an eye...